src/pymor/vectorarrays/numpy.py

```diff
@@ -48,6 +48,11 @@ def real(self):
     def imag(self):
         return NumpyVectorArray(self.data.imag.copy(), self.space)
 
+    def conj(self):
+        if np.isrealobj(self.data):
+            return self.copy()
+        return NumpyVectorArray(self.data.real.copy() - 1j * self.data.imag.copy(), self.space)
```

Comment:
Any reason for not using `np.conj`?  Also, we should ensure that for a non-complex array no unneccessary copy is made. (We can simply return `self.copy()` due to COW semantics of our VectorArrays.)